clients/go/internal/containersuite/containerSuite.go

```diff
@@ -57,6 +59,35 @@ func (s zeebeWaitStrategy) WaitUntilReady(ctx context.Context, target wait.Strat
 		return err
 	}
 
+	defer func() {
+		_ = zbClient.Close()
+	}()
+
+	finishedChan := make(chan error, 1)
+	go s.waitForTopology(finishedChan, zbClient)
```

Comment:
I wouldn't pass the channel into the method since I think it restricts the reusability of this method (you can only use it in a separate goroutine and you have to give it a channel). I'd either: a) make the method synchronous and have the lambda put the error value in the channel or b) make it asynchronous and self contained, by having it create the channel and go routine itself

Reply:
How would you time it out if we made synchronous?

Reply:
```go
	go func() {
		finishedChan <- s.waitForTopology(zbClient)
	}()
```
and waitForTopology would just synchronously return the error